indico/cli/user.py

```diff
@@ -217,6 +219,42 @@ def unblock(user_id):
         click.secho('Successfully unblocked user', fg='green')
 
 
+@cli.command()
+@click.argument('user_id', type=int)
+def anonymize(user_id):
+    """Anonymize a given user.
+
+    This will remove all personal data about the given user including
+    their name, affiliation and emails. Note that any data tied to events
+    such registrations will not be deleted.
+    """
+    user = User.get(user_id)
+    if user is None:
+        click.secho('This user does not exist', fg='red')
+        return
+    _print_user_info(user)
+    if click.confirm(click.style('Anonymize this user?', fg='yellow')):
+        user.first_name = '<anonymous>'
+        user.last_name = '<anonymous>'
+        user.title = UserTitle.none
+        user.affiliation = ''
+        user.affiliation_id = None
+        user.phone = ''
+        user.address = ''
+        user.picture = None
+        user.picture_metadata = None
+        user.picture_source = ProfilePictureSource.standard
+        user.favorite_users = set()
+        user.favorite_events = set()
+        user.favorite_categories = set()
+        user.identities = set()
+        # TODO: check what breaks when we delete all emails
```

Comment:
I think the system user also has no email so you can check for places where we check if a user is the system user and see if there's anything important